build.gradle

```diff
@@ -75,12 +75,14 @@ configure(subprojects) {
     }
 
     repositories {
+    	mavenLocal()
```

Comment:
Is this a mistake?

Reply:
No it is not. This is necessary for the workaround to add Maven project dependencies to Gradlle projects.